src/test_meta_srv.c

```diff
@@ -24,31 +24,30 @@
 
 /**
  * Performs the META test.
- * @param ctlsockfd Client control socket descriptor
+ * @param ctl Client control Connection
  * @param agent UNUSED Web100 agent used to track the connection
  * @param testOptions The test options
  * @param conn_options The connection options
  * @return 0 - success,
- *          >0 - error code.
- *          Error codes:
- *           -1 - Cannot write message header information while attempting to send
- *           		 TEST_START message
- *          -2 - Cannot write message data while attempting to send
- *           		 TEST_START message
- *          1 - Message reception errors/inconsistencies
- *			2 - Unexpected message type received/no message received due to timeout
- *			3 - Received message is invalid
- *			4 - Invalid data format in received message
+ *         Error codes:
+ *          -2 - Cannot write message data while attempting to send TEST_START
```

Comment:
Dropped the word `message` after `TEST_START`. I'm fine with cutting it, but if we do we should cut it for `-1` as well